src/components/dev-hub/learn-tabs.js

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { Tabs as LeafyTabs, Tab } from '@leafygreen-ui/tabs';
+import Button from './button';
+
+const TabButtonStyle = () => css`
+    border-radius: 0;
+    :active,
+    :hover,
+    :focus {
+        color: ${({ theme }) => theme.colorMap.darkGreen};
+        :after {
+            background-color: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const StyledTabButton = styled(Button)`
+    ${TabButtonStyle}
+`;
+
+const TabsStyle = () => css`
+    @media (min-width: 1048px) {
+        button:first-child {
+            margin-right: 73%;
+        }
+    }
+    button[aria-selected='true'] {
+        color: ${({ theme }) => theme.colorMap.darkGreen};
+        :after {
+            background-color: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const StyledTabs = styled(LeafyTabs)`
+    ${TabsStyle}
+`;
```

Comment:
```suggestion
const TabsStyle = (theme) => css`
    @media (min-width: 1048px) {
        button:first-child {
            margin-right: 73%;
        }
    }
    button[aria-selected='true'] {
        color: ${theme.colorMap.darkGreen};
        :after {
            background-color: ${theme.colorMap.darkGreen};
        }
    }
`;

const StyledTabs = styled(LeafyTabs)`
    ${({theme}) => TabsStyle(theme)};
`;
```

`css` does not use `theme` in the same way that `styled` does, so we explicitly have to pass it if we want to use `css`. Another thought is could we just put all of the style into `StyledTabs` like the below?

```
const StyledTabs = styled(LeafyTabs)`
        @media (min-width: 1048px) {
        button:first-child {
            margin-right: 73%;
        }
    }
    button[aria-selected='true'] {
        color: ${({ theme }) => theme.colorMap.darkGreen};
        :after {
            background-color: ${({ theme }) => theme.colorMap.darkGreen};
        }
    }
`;
```

That will work since `theme` is always passed to `styled`